lib/qtermwidget.cpp

```diff
@@ -306,6 +306,8 @@ void QTermWidget::init(int startnow)
 
 QTermWidget::~QTermWidget()
 {
+    delete ColorSchemeManager::instance();
```

Comment:
This breaks the remaining `QTermWidget`s because they still use the singleton (which just got deleted).